src/components/Searchbar.js

```diff
@@ -1,15 +1,161 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
 import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
 import constants from '../styling/Constants';
 import { TextInput, DateInput, StyledButton } from '../components';
+import { useHistory } from 'react-router-dom';
```

Comment:
nitpick but some managers will ask for this: please order the imports in alphabetical order

Reply:
updated import order now -- kind of a dumb point of uncertainty but still not sure if i ordered them right LOL, just let me know if not

Reply:
where is it?

Reply:
not dumb! thanks for asking
Here's how I would order
```
import React, { useEffect, useState } from 'react';
\\new line here
import { makeStyles } from '@material-ui/core/styles';
import { TextInput, DateInput, StyledButton } from '../components';
import { useHistory } from 'react-router-dom';
\\new line here
import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
import constants from '../styling/Constants';
```